content/pages/5_roadmap_de.md

```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
 <br />
 Hier unten finden Sie die Roadmap von GeoMapFish, aber auch die Links zu den Release Notes der verschiedenen Versionen.
 
-Letztes Update der Seite: **23.09.2020**
+Letztes Update der Seite: **10.06.2021**
+
+Für eine komplette Roadmap bis 2024, sehen dieser Dokument: [GMF_2024_Roadmap.pdf](/documents/meetings/2021-06-08/GMF_2024_Roadmap.pdf)
+
+## Version Snook (2.7 LTR)
+
+Nächste Version.
+
+* Entwicklungen: **AnfangSeptember 2021**
+* Geschätztes Freigabedatum: **Summer 2022**
+
+Diese Version habe keine neuen funktionalitäten. Es ist nur eine technische Migration, und das Ziel ist GeoMapFish ohne AngularJS zu bereiten, weil AngularJs ist nicht mehr seit ende 2021 unterstützt.
```

Comment:
Shouldn't we talk about the two new functionalities presented today:

- WFS3
- VectorTiles